benchmark_v2/groupby_benchmark.py

```diff
@@ -45,11 +37,19 @@ def generate_arrays(dtype, numArrays):
 @pytest.mark.parametrize("dtype", TYPES)
 def bench_groupby(benchmark, numArrays, dtype):
     if dtype in pytest.dtype:
-        arrays, totalbytes = generate_arrays(dtype, numArrays)
-        benchmark.pedantic(ak.GroupBy, args=[arrays], rounds=pytest.trials)
+        N = 10**4 if pytest.correctness_only else pytest.prob_size * ak.get_config()["numLocales"]
```

Comment:
Makes me wonder whether we should query `ak.get_config()` at server startup and stash it in `pytest.ak_config` or some such. (And clear it up when the server is shutdown.)

This is independent of this PR of course.